src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -38,7 +38,7 @@ export function WhaleSightingViewer({loggedIn, isAdminPage} : WhaleSightingViewe
 				<WhaleSightingFilters setSightings={setSightings}/>
 			</div>
 			<div className="whale-sighting-sort">Sort to go here</div>
-			<SightingList pageNum={page} sightings={sightings} loggedIn={loggedIn} isAdmin={isAdminPage}/>
+				<SightingList pageNum={page} sightings={sightings} loggedIn={loggedIn} isAdmin={isAdminPage}/>
```

Comment:
Can we put the props on new lines to make this easier to read?
```suggestion
				<SightingList 
				    pageNum={page} 
				    sightings={sightings} 
				    loggedIn={loggedIn} 
				    isAdmin={isAdminPage}/>
```